source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -772,6 +772,21 @@ InjaTransformer::InjaTransformer(const TransformationTemplate &transformation,
     merged_extractors_to_body_ = true;
     break;
   }
+  case TransformationTemplate::kMergeJsonKeys: {
+    if (transformation.parse_body_behavior() == TransformationTemplate::DontParse) {
+      throw EnvoyException("MergeJsonKeys requires parsing the body");
```

Comment:
https://github.com/envoyproxy/envoy-mobile/issues/176
Any way we can also avoid an exception here and just return out?
Not a big deal if not as it seems to mostly be a build sizing thing rather than a true runtime performance thing

Reply:
What do you mean `just return out`? We specifically want to Let the user know this isn't supported, we NEED to parse the body for this functionality to work

Reply:
Im dumb I thought that this was on the request path. Disregard